lib/WP_Auth0_LoginManager.php

```diff
@@ -578,8 +588,6 @@ public function auth0_singlelogout_footer() {
 
 	/**
 	 * End the PHP session.
-	 *
-	 * TODO: Remove, not necessary!
```

Comment:
this method is necessary now or what?

Reply:
Should be deprecated, added that comment below 👇